src/fides/api/ops/models/messaging.py

```diff
@@ -144,3 +158,44 @@ def set_secrets(
 
         self.secrets = messaging_secrets
         self.save(db=db)
+
+    @classmethod
+    def get_active_default(cls, db: Session) -> Optional[MessagingConfig]:
```

Comment:
this is great to have, and I think something we can re-use throughout the codebase to help validate configs prior to attempting actual messaging sends during run-time

Reply:
oh good point. so far i'd only updated the app code to use the `config_proxy` to determine the notification service type, e.g. https://github.com/ethyca/fides/blob/main/src/fides/api/ops/service/_verification.py#L37, but i think maybe it's a better idea to rely on this utility method in all places, as i think you're suggesting. just consolidates our code a bit...

let me know if that's what you're suggesting

Reply:
We can follow-up with this improvement. Can you create a new tech-debt ticket for using this utility method where we can?

Reply:
created https://github.com/ethyca/fides/issues/2663